src/components/Section.astro

```diff
@@ -0,0 +1,41 @@
+---
+interface Props {
+    dark?: boolean;
+}
+
+const { dark = false } = Astro.props;
+---
+
+<section class={dark ? 'dark' : ''}>
```

Comment:
**Suggestion:** It would be nice to be able to add an `aria-labelledby` to the `SectionTitle`. That can be another PR, and I'm happy to take that.